introduction.md

```diff
@@ -1,3 +1,9 @@
 Hi, my name is Jonathan and I am working on the course Mastering GitHub from CodeSchool.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?